File: prime.py
import sys
import math

max = int(sys.argv[1])
print(max)

def isprime(num):
    count = 2
    while count < num:
        d = num/count
        if d - math.floor(d) == 0.0:
            return False
        count+=1
    return True

p = 1
for i in range(1, max):
    if isprime(i):
        print(p, i)
        p+=1

File: prime.js
var max = parseInt(process.argv[2])

isprime = function(num){
    var count = 2
    while (count < num){
        var d = num/count
        if (d - Math.floor(d) == 0.0){
            return false
        }
        count+=1
      }
    return true
}

p = 1
for(var i = 0; i <= max; i++){
    if (isprime(i)){
        console.log(p, i)
        p+=1
    }
}
